### PR TITLE
Added configfile and opts

### PR DESCRIPTION
**Pull request by lqb**

### CODE REVIEW

File: wgked.conf.example [hunk 1]
```diff
@@ -0,0 +1,14 @@
+segments:
```
Comment (mweinelt) [inline wgked.conf.example:1]: We would say `domains` in Gluon terminiology btw.

File: wgked [hunk 2]
```diff
@@ -3,27 +3,14 @@ from flask import Flask, render_template, jsonify, request
...
-'ffmuc_uml_nord',
-'ffmuc_uml_ost',
-'ffmuc_uml_sued',
-'ffmuc_uml_west',
-'ffmuc_welt'
-]
+app = Flask(__name__)
 
 def is_valid_wg_pubkey(pubkey):
+  WG_PUBKEY_PATTERN = re.compile("^[A-Za-z0-9+/]{42}[A|E|I|M|Q|U|Y|c|g|k|o|s|w|4|8|0]{1}=$")
```
Comment (mweinelt) [inline wgked:13]: This makes compiling the regex useless as it will not be reused between validation calls.

Comment (awlx) [reply wgked:13]: @lqb

Comment (mweinelt) [reply wgked:13]: Fixed in 5e453f483a6ac30d78b2578b7cea55435eb01a56

**Pull request merged**